docs/translations/translations-in-PHP-files.md

```diff
@@ -0,0 +1,202 @@
+# Translations in PHP files
```

Comment:
Again, if this document is targeting WooCommerce Blocks and its consumers/contributors, then I think it make sense to also add that we load translation from two places, first from WooCommerce Blocks, then from WooCommerce Core if we fail to find the translation in blocks.

For PHP, this happens here
https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/b99bb6e28300ffddf05c3108ec9f76a5537373cd/woocommerce-gutenberg-products-block.php#L240-L263

For JS, this happens here

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/b99bb6e28300ffddf05c3108ec9f76a5537373cd/woocommerce-gutenberg-products-block.php#L154-L239

Reply:
Thanks for pointing this out. I've added a new doc to the translation section.